### PR TITLE
travis: apache2 xsendfile installation

### PR DESCRIPTION
**Pull request by hachreak**
* FIX Adds apache2 xsendfile package to installation process.
  (addresses #2857)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ before_install:
   - "touch .travis-release-requirements.txt"
 
 install:
-  - "sudo apt-get -qy install apache2 libapache2-mod-wsgi ssl-cert poppler-utils git subversion nodejs --fix-missing"
+  - "sudo apt-get -qy install apache2 libapache2-mod-wsgi libapache2-mod-xsendfile ssl-cert poppler-utils git subversion nodejs --fix-missing"
```
Comment (jirikuncar) [inline .travis.yml]: Is it necessary to enable it (`sudo a2enmod xsendfile`) too or it's enabled by defauilt?

Comment (hachreak) [inline .travis.yml]: `Module xsendfile already enabled`
I removed the `a2enmod`

**Comment by hachreak** [comment]
```
/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/nose/plugins/manager.py:395: RuntimeWarning: Unable to load plugin pylons = pylons.test:PylonsPlugin: No module named pylons.test
```

@jirikuncar  Do we need to add this module on setup.py?

**Comment by jacquerie** [comment]
@hachreak: I think that's an instance of this bug: https://github.com/nose-devs/nose/issues/874.

In fact, removing the second stanza in `lib/python2.7/site-packages/fixture-1.5.1-py2.7.egg-info/entry_points.txt` removes this warning.

**Pull request merged**